Documentation/guides/AndroidAssetPacks.md

```diff
@@ -0,0 +1,176 @@
+# Android Asset Packs
+
+Google Android began supporting splitting up the app package into multiple
+packs with the introduction of the `aab` package format. This format allows
+the developer to split the app up into multiple `packs`. Each `pack` can be
+downloaded to the device either at install time or on demand. This allows
+application developers to save space and install time by only installing
+the required parts of the app initially. Then installing other `packs`
+as required.
+
+There are two types of `pack`. The first is a `Feature` pack, this type
+of pack contains code and other resources. Code in these types of `pack`
+can be launched via the `StartActivity` API call. At this time due to
+various constraints .NET Android cannot support `Feature` packs.
+
+The second type of `pack` is the `Asset` pack. This type of pack ONLY
+contains `AndroidAsset` items. It CANNOT contain any code or other
+resources. This type of `feature` pack can be installed at install-time,
```

Comment:
The use of "feature" here may be confused with the previous paragraph's mention of `Feature` packs.  Perhaps this sentence should instead say "`Asset` packs can be installed at …"?